src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -610,4 +610,7 @@
   <data name="The_first_word_0_must_begin_with_a_lower_case_character" xml:space="preserve">
     <value>The first word, '{0}', must begin with a lower case character</value>
   </data>
+  <data name="File_size_exceeds_maximum_allowed_size_0" xml:space="preserve">
+    <value>File size exceeds maximum allowed size: {0}</value>
```

Comment:
We're past loc cutoff, so it might be better to just use a hardcoded message in RTW, and update later.  Not sure though.